### PR TITLE
Fix SVG in Safari

### PR DESCRIPTION
**Pull request by richardgoater**

### CODE REVIEW

File: metaspace/webapp/src/components/PrimaryIcon.vue [hunk 1]
```diff
@@ -15,13 +15,16 @@ export default defineComponent({
   },
 })
 </script>
-<style scoped>
+<style scoped lang="postcss">
```
Comment (LachlanStuart) [inline metaspace/webapp/src/components/PrimaryIcon.vue:18]: Why postcss? Can this be scss?

File: metaspace/webapp/src/components/PrimaryIcon.vue [hunk 2]
```diff
@@ -15,13 +15,16 @@ export default defineComponent({
...
+i >>> svg {
+  width: 100%;
+}
 i >>> .primary,
 i.inverse >>> .secondary {
-  @apply text-blue-400;
+  @apply fill-current text-blue-400;
```
Comment (LachlanStuart) [inline metaspace/webapp/src/components/PrimaryIcon.vue:24]: Are the `fill-current` changes a necessary part of the fix? I'm not asking for them to be reverted, just trying to understand what happened with Safari.

**Review by LachlanStuart — APPROVED** [review]
Fine to merge, but I'd prefer it if the `postcss` blocks were `scss`, just to minimize tech stack dilution.

File: metaspace/webapp/src/components/PrimaryIcon.vue (hunk 2 above)
Comment (LachlanStuart) [reply metaspace/webapp/src/components/PrimaryIcon.vue:24]: Nevermind, I just saw the commit message about this. Makes sense now.

File: metaspace/webapp/src/components/PrimaryIcon.vue (hunk 1 above)
Comment (richardgoater) [reply metaspace/webapp/src/components/PrimaryIcon.vue:18]: this was a blunt method of fixing linting and syntax highlighting, I can remove it.

**Pull request merged**